purchases/src/main/kotlin/com/revenuecat/purchases/google/usecase/QueryProductDetailsUseCase.kt

```diff
@@ -23,20 +22,21 @@ import java.util.Date
 import java.util.concurrent.atomic.AtomicBoolean
 import kotlin.time.Duration
 
-internal data class QueryProductDetailsUseCaseParams(
```

Comment:
Hmm it's sad we won't get the `data class` anymore with its benefits... I wonder if we should make `UseCaseParams` an interface instead of a class. Then, these data classes just need to add the `override` for that property I think? We would need to create a new class for `GetBillingConfigUseCase` since that didn't have parameters before, but I think that's preferrable. Thoughts?

Reply:
perfect, I will do that